README.md

```diff
@@ -50,247 +48,106 @@ const results = sass.renderSync({
 });
 ```
 
-## API
-
-<a name="module_node-sass-extra"></a>
-
-## node-sass-extra
-The Node API for `node-sass-extra`.
+## Options
 
+|Name|Type|Description|
+|----|----|-----------|
+|data|`string` \| `string[]`|String(s) to be compiled.|
+|file|`string` \| `string[]`|File(s) to be compiled; can be a file path or a glob pattern.|
+|output|`string` \| `function`|The output destination; if provided, files WILL be written to disk. Can be a file path, a directory, or a callback that returns a file path or directory. Must be/return a file path when paried with `data`.|
```

Comment:
"... a file path when _paired_ with `data`."

One more "paried" below, too.

Reply:
Good catch, ill get this updated and and merged in.